k8s/overlays/develop/secrets.enc.yaml

```diff
@@ -0,0 +1,17 @@
+  POSTGRES_PASSWORD: wfVvDymiXz8r/Ur9gGQd4t5pUx4dWQgunOGHug/xpoqlToZNVNSvh8vGbV/S2CFX.gqk4DTt7D8yR1w3n.cXfcfGGNnM3pVu+HSCblxg==
+  POSTGRES_USER: O4700lcP6uVgKeSBPFEtUVz7GFYeTtDPIxXXroEtlTwjr4lQ+Wwg5yFwv7x599rx.2U5gQQ/NnQvvzGzD.pPM+5H6kkomS5qcbwHzmvA==
+  RABBITMQ_PASSWORD: Z0sER01daoVqEM05uL3o7jqKtpaeOeL96CoV1QelESFDANVLwQAg2xew5ewVYh3F.trifLWtxaVBYhZ74.wNqU17k0KPrTIkzJrq/iTw==
+  RABBITMQ_USER: /uC+Z5QCsAJiVltFnkmOAU9FOhoDNa0OlNZZneuZ6Z8DxuByfqyiRwIOFyiH/EOR.X2VZDLg35IBM+n4g.jrzwqyb4pZMhhz/opubSxA==
+kind: Secret
+metadata:
+  name: onboarding-secrets
+type: Opaque
+# kubesec:v:3
+# kubesec:pgp:8D3D18535FBB11BD7652675B5D501B55008EF1AD:LS0tLS1CRUdJTiBQR1AgTUVTU0FHRS0tLS0tCgpoUUlNQTZ3VFZia25uUEhVQVJBQXVoa1lGSkV6MzQzWUNWTlNyZ29sNE9LS3J3WFZMUlB0L01FcWIrYkhHSHhPCmtUOE5rV0ZLakI0aG1VOUlYU3FPbGUzbkcyTUkwam1UUWlLUE9PNXo0VW1QRkNLVTgyYlNvOFJxOEdoSWdKS2MKREZsbTE0SGM1TlFzSzFzeFpXOXpvWnp4SURaS3NRZlpEM2NXM000bGVyaGJnQXBRREVhdG9MRmU5TnpCcElOUQpja0dBbldXVHJqd00vZzhZQ1c2c1FkaGcrZmUvQjlmMVlWckZ6cm1qa0tSWTgrWndSQjRVSUdyWHNyODI0SzZXCjFwN0Y0QUoyejEwRjEyZ3M1dkNqSkt1WXJLcVNBWDVONmwycDN4QzRZV25zNU9sMDl4VnZ3VFdLVy8weFRXbzUKSU44aWpBa2MwSExNWWI3T3lUVDFEM1BJODBnNEhEQlBRdzdSMGhLNmJlR3R5c21oU0xzR3NBQm5qZEpRaEZrLwpOZ0Ixa2w1T2k5QWxsamlZQURoYW9wMUxGVlR1NFMxN2RtNXdObXRkRndPZ3F4bDNlQk5uZHArM1RBYStHalA3Ci82NWlTQTgzdmFaZC9panlJanFSVGpSemFYeEdlaDVVTStLbVp4N2FxTnV3b1hkUW9GS0ZmbGVVUWVmcjhpYWQKOUV4WFJXRHU1bVpnR3V0a05Qbld4OVYxcmNRNHlHVnI2UmJqc25hUGJ1S3BqL1VvSUVURC9WNXJGK0dYTVVFRQpxMGFacTJkYTFrQzRKWGRXam5WNEpadjdwdFlLYktvcUZqV2UwWkE0SkdnM2FtejZ2S0dlVUtsYzJ4U3YxUHhTCm04K05lM2VHQnRndEdGS0pPTXpoTDUwelgzQlJ0ZnlETllDaTdJVzlKRVdTVXhEd21xYUhINGxxYlZrVDlYRFUKNlFFSkFoRENNdzNqNUZFdmxsUmcxNDNkQ0hSS1dZL3lCNG9WMkN1VlJqRkVEQWZmTTAxYmsrUWlFSjZTUFNVbgpOMFNETGkvQ2VlREtyQkdFd3RyM3FBWmdCaUdMWC9SZDJXa1I5WXdEcW5WODg3WStOUExNdjNWem1PQTZRaXI4Cm84dVQreTloK1ZLUkRjK2FpcDE5RkZjNEhKMUZ0Q0dNTnJzUXA5SUM3amhnWHJ0a0xsMUp3Y0EwcFNmOW5pMDkKaVZ4T0VMblk0VktlanFrNXRaVXB2VlJIZUJiSmVJL1lyK2VUQXpDbDU4S0E1WnRDZ3JYeEYvOFVOTlNpNWJRWApTbHAveXBCaDFPU2hRQVl1aFhNODB0K3FHa2dzaUNkdCtsUzI5TkFaTjdlTFZGMmphWHUybTF0MjFTZFplZkxXCjNmbm1PVmdqVTdmbVkxeW0vZHdYZ0lMTFc3Nkc0NTFnRnhtdkJ3RHVzaDBRMkh6VFk3dHg1cDRUYmxtQUhJazcKVHhseUt5MjVldzZVOUdRYVYzTGI2OXpOcG91cFp6VDRNUGVwRStNZXhzVlUvdWRhNEpwMnRVeVJwTnFKRHRmcgpuY3FFcDI2ejNVeWFNRWxVa1lVY0loaXdlbXVRZThXWEZBLzVGWW1ac0hyRWRVc3V4TmlqT29XR2NzbDNYL1FjCi91UGVqRGxnWE90U2kxVG1aTzhBOE1WdmlLMnRMNFE1NzZCWUl0aTNoaGwxWE0zaW1FUmd4RFV6a3U4T3dUOUgKWUx2bngyTnM1c0NVblZPUlVxcTBaNDJ1V3JBSXQ3WUNTQUcxMEptckFROXlHb01aeThLRy90dUdYL1A1YW5UKwpLUkY1RjFaMFZGN2FKMzdhVXFWLy81NlQxRXNtQWpaVkJQbWdDZjRxMnNTQXRsZlMrSm1iRkJSclg1ODdzRDdBCmxqa1c4VXpDSExhVHlIbU9LdXIrSHV5QnoxUHlPUlZ0Ym9PTExObGY2MWpkSUNkSmplY2EyUFpNZ0w0ZExlelIKSFhPZktzK01KNGZTMUpNU2ZjcEYwZlhIbHZlVzJ0TTlIeUQ1MkloMldzYUt4bkJWekVERk95L1FQaEs3WE9KUwpDUVVEQUVBanVsaHdSeC9UcEhJNU45RmV4SXFScnVMQkhiMkRyN0UxMGNKamdHcWptRjB4QXhpOVNzd1BDYUEyCkw1blExcUtqeHNLY1ZqTTNaTWN4bWwydUNycDNSWXZ5Cj1ZKy9yCi0tLS0tRU5EIFBHUCBNRVNTQUdFLS0tLS0K
```

Comment:
This will be referencing your local GPG key so they won't be able to decrypt it to edit it until they've re-encrypted it with their key, which could be a catch-22. In other projects we're able to commit encrypted secrets because they reference a key stored in KMS that we all can use locally or in CI.

For this example, it might be better to commit the decrypted secret, e.g. `secret.yaml.example` since we're not working with any live environment, and make a note that usually we'd commit the encrypted secret that's edited in-place.

Reply:
Update: I thought this was on the software onboarding repo and didn't realise this was the personal project.